makefiles/youtube-dl.mk

```diff
@@ -28,11 +28,11 @@ endif
 youtube-dl-package: youtube-dl-stage
 	# youtube-dl.mk Package Structure
 	rm -rf $(BUILD_DIST)/youtube-dl
-	mkdir -p $(BUILD_DIST)/youtube-dl
-
-	# youtube-dl.mk Prep youtube-dl
 	cp -a $(BUILD_STAGE)/youtube-dl $(BUILD_DIST)
 
+	# youtube-dl.mk Sign
+	$(call SIGN,youtube-dl,general.xml)
```

Comment:
Why are you signing the package, it's a Python program?

Reply:
I've always done this. ``youtube-dl`` is an executable. I can remove it if it's unnecessary.